packages/ui/src/atoms/Link/Link.test.tsx

```diff
@@ -0,0 +1,51 @@
+import { render } from '@testing-library/react'
+import { axe } from 'jest-axe'
+import React from 'react'
+
+import type { LinkProps } from '.'
+import LinkComponent from '.'
+
+const TestLink = ({ href = '/', testId }: LinkProps<'a'>) => {
+  return (
+    <LinkComponent testId={testId} href={href}>
+      Link
+    </LinkComponent>
+  )
+}
+
+describe('Link', () => {
+  describe('Data Attributes', () => {
+    it('`Link` should have `data-store-link` attribute', () => {
```

Comment:
[Low priority] Usually, the test runners compose the final test "name" with all parents, so it's not needed to repeat "Link" here. I imagine they'd output something like: "Link Data Attributes 'Link' should have…".